File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum = 0;
        int nSum = 0;
        for (int i = 0; i < nums.length; i++) {
            sum += nums[i];
            nSum += i;
        }
        return nSum + nums.length - sum;
    }
}

File: 0268-missing-number/0268-missing-number.kt
class Solution {
    fun missingNumber(nums: IntArray): Int {
        var sum = 0;
        var sumN = 0;
        for (i in 0..nums.size-1) {
            sum += nums[i]
            sumN += i
        }
        return sumN + nums.size - sum;
    }
}